Components/Paypal/AddressValidator.php

```diff
@@ -35,6 +35,7 @@ public function __construct(AddressValidatorInterface $innerValidator, DIContain
     public function validate(Address $address)
     {
         if (!$this->container->get('front')->Request()
+            || $this->container->get('front')->Request() === null
```

Comment:
@mitelg : Sorry, i did not compare the version of this plugin. In our case the first line is "if (!$this->container->get('front')". Seems to be fixed already.